src/components/Video/VideoTracksScrubber.vue

```diff
@@ -34,18 +35,27 @@
   </div>
 </template>
 
-<script>
+<script lang="ts">
+// eslint-disable-next-line no-unused-vars
+import { Track } from "../../api/Recording.api";
```

Comment:
Does eslint not look in TS type signatures?

Reply:
Turns out I needed to use the "@typescript-eslint/no-unused-vars": version of this lint in .eslintrc.js